### ISSUE TITLE
[bug] Unhandled Exception: FormatException: Invalid radix-10 number

### ISSUE THREAD

**Issue by ihipop** [reqable/reqable-app#339]
**Describe the bug**
Unhandled Exception: FormatException: Invalid radix-10 number

**To Reproduce**
Just Launch reqable-app from terminal
The App will run, but the error is there

```
(process:150825): GLib-GIO-CRITICAL **: 18:38:20.007: g_application_set_application_id: assertion 'application_id == NULL || g_application_id_is_valid (application_id)' failed
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: FormatException: Invalid radix-10 number (at character 1)
null
^

*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
pid: 150825, tid: 140182353123008, name io.flutter.ui
os: linux arch: x64 comp: no sim: no
build_id: '4d7b26e9cfc072933bfaaa0f2e7ff6d5'
isolate_dso_base: 7f7ebf5cc000, vm_dso_base: 7f7ebf5cc000
isolate_instructions: 7f7ebf9ed040, vm_instructions: 7f7ebf9e8000
    #00 abs 00007f7ebfa0855a virt 000000000043c55a _kDartIsolateSnapshotInstructions+0x1b51a
    #01 abs 00007f7ebfa07948 virt 000000000043b948 _kDartIsolateSnapshotInstructions+0x1a908
    #02 abs 00007f7ebfa07814 virt 000000000043b814 _kDartIsolateSnapshotInstructions+0x1a7d4
    #03 abs 00007f7ebfa07339 virt 000000000043b339 _kDartIsolateSnapshotInstructions+0x1a2f9
    #04 abs 00007f7ebfedf507 virt 0000000000913507 _kDartIsolateSnapshotInstructions+0x4f24c7
    #05 abs 00007f7ebffc45f7 virt 00000000009f85f7 _kDartIsolateSnapshotInstructions+0x5d75b7
    #06 abs 00007f7ebffc4411 virt 00000000009f8411 _kDartIsolateSnapshotInstructions+0x5d73d1
    #07 abs 00007f7ec03ea710 virt 0000000000e1e710 _kDartIsolateSnapshotInstructions+0x9fd6d0
    #08 abs 00007f7ec033fb3d virt 0000000000d73b3d _kDartIsolateSnapshotInstructions+0x952afd
    #09 abs 00007f7ebf9fe46f virt 000000000043246f _kDartIsolateSnapshotInstructions+0x1142f
    #10 abs 00007f7ebf9fe6b0 virt 00000000004326b0 _kDartIsolateSnapshotInstructions+0x11670
    #11 abs 00007f7ec0348b97 virt 0000000000d7cb97 _kDartIsolateSnapshotInstructions+0x95bb57
    #12 abs 00007f7ebf9fdc2e virt 0000000000431c2e _kDartIsolateSnapshotInstructions+0x10bee
    #13 abs 00007f7ebf9fea41 virt 0000000000432a41 _kDartIsolateSnapshotInstructions+0x11a01
    #14 abs 00007f7ebfa15793 virt 0000000000449793 _kDartIsolateSnapshotInstructions+0x28753
    #15 abs 00007f7ebf9f6bc2 virt 000000000042abc2 _kDartIsolateSnapshotInstructions+0x9b82
    #16 abs 00007f7ec0346499 virt 0000000000d7a499 _kDartIsolateSnapshotInstructions+0x959459
    #17 abs 00007f7ebf9f6b2e virt 000000000042ab2e _kDartIsolateSnapshotInstructions+0x9aee

```

**Expected behavior**
No Error

**Screenshots**
![image](https://github.com/reqable/reqable-app/assets/423077/6fd468f4-6621-4437-81b8-55b5ab74bf89)

**Information**
 - Platform: Linux
 - OS: Manjaro Linux 23.1.1 
 - Arch:  x86_64
 - App Version: 2.1.1

**Comment by MegatronKing** [comment]
@ihipop Thanks for submitting the issue. It seems that this bug is caused by the proxy port conflict. We will fix it in the next release. The workaround is to find the used 9000 port and close it, then restart reqable again.

**Comment by MegatronKing** [comment]
Fixed on v2.2.0.
